bin/python/combine_reports_from_date_range.py

```diff
@@ -88,12 +104,15 @@ def main(device, start=None, end=None, gcp=None, reports_dir=os.path.join('inst'
                 aggregate['tests'][test][result] += 1
                 aggregate['categories'][category][result] += 1
     result_str = _render_results(aggregate)
-    reports_path = os.path.join(reports_dir, 'combo_%s_(%s,%s).md' % (device, start, end))
+    start_stamp = _iso_to_fname(start)
+    end_stamp = _iso_to_fname(end)
+    reports_path = os.path.join(reports_dir, 'combo_%s_%s_%s.md' % (device, start_stamp, end_stamp))
     with open(reports_path, 'w') as report_file:
         report_file.write('# Combined Results\n')
-        report_file.write('Device: %s Start: %s End: %s\n' % (device, start, end))
+        report_file.write('Device: %s\nStart: %s\nEnd: %s\n' % (device, start, end))
         report_file.write(result_str)
     LOGGER.info('Report written to %s' % reports_path)
+    assert not count or count == len(json_reports), 'Did not find expected %d reports' % count
```

Comment:
I don't really think the assertion is necessary. It's hard to know how many reports you have if you specify the device, start and end.